apps/vmq_server/test/vmq_queue_SUITE.erl

```diff
@@ -236,6 +240,58 @@ queue_persistent_client_expiration_test(_) ->
     not_found = vmq_queue_sup_sup:get_queue_pid(SubscriberId),
     {ok, []} = vmq_lvldb_store:msg_store_find(SubscriberId).
 
+queue_force_disconnect_test(_) ->
+    Parent = self(),
+    SubscriberId = {"", <<"force-client-disconnect">>},
+    QueueOpts = maps:merge(vmq_queue:default_opts(), #{clean_session => false,
+                                                       max_offline_messages => 1000,
```

Comment:
Should this be a hardcoded 1000 or use the User's setting?

Reply:
you're right, for this test case it really doesn't matter.